app/controllers/practices_controller.rb

```diff
@@ -186,10 +181,6 @@ def search
 
   end
 
-  def planning_checklist
-    @facilities_data = facilities_json
-  end
-
   # GET /practices/1/committed
   def committed
```

Comment:
we also don't need this method along with the commit method on line 200

Reply:
If we get rid of the commit method, we should get rid of the route in `routes.rb` as well.

Reply:
Got it.